javascript-engine/src/utils.ts

```diff
@@ -0,0 +1,24 @@
+type EngineResponse = {
```

Comment:
Personal feeling but "utils" is where good project structure goes to die. This file looks like it has a pretty focused intent. Can we name it for what it does? I want to say this is doing WASM interop so.... what about wasm-interop?

Reply:
Are you saying I should I build an util in utils to better organize our utils by util category?

Sorry.

I 100% agree but naming is hard. I like your suggestion 👍 https://github.com/Unleash/yggdrasil/pull/226/commits/bc84e4aff483fbb39a4de9ec0047d8567d0c0fbb